rts/Game/Camera/RotOverheadController.cpp

```diff
@@ -44,10 +46,11 @@ void CRotOverheadController::KeyMove(float3 move)
 
 void CRotOverheadController::MouseMove(float3 move)
 {
-	camera->SetRotY(camera->GetRot().y + mouseScale * move.x);
-	camera->SetRotX(camera->GetRot().x + mouseScale * move.y * move.z);
-	camera->SetRotX(Clamp(camera->GetRot().x, math::PI * 0.4999f, math::PI * 0.9999f));
-	dir = camera->GetDir();
+	auto rot = CCamera::GetRotFromDir(dir);
```

Comment:
Is this called on purpose instead of `camera->GetRot()`, if so can you comment the rationale?